### ISSUE TITLE
Webhosting with 123-REG installation issues

### ISSUE THREAD

**Issue by afacciponti** [kiswa/TaskBoard#419]
Hi, Im having issues settting up taskboard with my hosting provider.

I have read and followed many of the posts but obviously still doing something wrong.

My hosting package type is linux and they use mysql. Php 5.5 is currently being used and i have set up other programs to work sucessfully in the past.

I have created an sql data base.

I have created a folder and copied the TaskBoard-master.zip file and unpacked it to it desired loaction renaming the folder to taskboard.

I also created a config.php file and configured and installed it as stated in one of the boards.

When I browse to the loaction of the taskboard it displays the login screen but refuses to accept the admin admin login no error message just jumps and does not go further. If I Chmod the api folder to 777 then it excepts the admin admin login but in settings it will not allow me to change the passwords.

Im locked out of this facility it seems that the database is not been created.
First should i be using 777 to unlock the api writability is this safe? second im sure i have set the config.php file correctly and placed it in the taskboard root folder but im sure i must be missing something.

Im still learning so its very possible i have missed a step.

has any one else got this working on 123-reg

Many thanks in advance 

Antonio

**Comment by afacciponti** [comment]
Ok just reading some more how do i run ./build/composer.phar install from a hosted website? Answers on a post card in plain english please :-)

Antonio

**Comment by afacciponti** [comment]
Can I use something like cyberduck to sent the command? If so do I need to just type ./build/composer.phar install or the full path?

**Comment by kiswa** [comment]
The how would depend upon your web host.

Simplest solution; run it locally and copy the files to your host.

**Comment by afacciponti** [comment]
Ok Thank you, It may take a few days bfore i get the oppertunity to do this. Thank you for your response.

**Comment by kiswa** [comment]
Of course. Let me know if you get it working, or if you need other help.

**Comment by afacciponti** [comment]
Hello,
Thank you for you initial help trying to set this up became something of a time consuming mission so I have now moved over to another Kanboard application that was simpler to install and did not require the additional steps. Thank you again.
